wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -2,30 +2,21 @@
     <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ site_no }}" data-latitude="{{ latitude }}" data-longitude="{{ longitude }}">
         <div class="loading-indicator-container"></div>
         <div class="graph-container"></div>
+        <div class="provisional-data-statement">
+            <p>
+                <span class="usa-tag">Important</span>
+                data may be provisional - <a class="usa-link" href="/provisional-data-statement">learn more</a>
```

Comment:
This won't work on deployment for the href tag.  You should use the Flask/Jinja function url_for. So it would look like: 
```
href="{{ url_for('provisional_data_statement') }}"
```
The string 'provisional_data_statement' is the name of the view

Reply:
Oh, great to know. Thanks for pointing that out.